File: CS/MsaglHelpers/Converter.cs
using MsaglPoint = Microsoft.Msagl.Core.Geometry.Point;
using System.Windows;

namespace MsaglHelpers {
    public static class Converter {
        public static Point MsaglPointToPointConvert(this MsaglPoint msaglPoint) {
            return new Point(msaglPoint.X, msaglPoint.Y);
        }
    }
}

File: VB/MsaglHelpers/Converter.vb
Imports MsaglPoint = Microsoft.Msagl.Core.Geometry.Point
Imports System.Windows

Namespace MsaglHelpers
    Public Module Converter
        <System.Runtime.CompilerServices.Extension> _
        Public Function MsaglPointToPointConvert(ByVal msaglPoint As MsaglPoint) As Point
            Return New Point(msaglPoint.X, msaglPoint.Y)
        End Function
    End Module
End Namespace